chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -125,6 +127,58 @@ pub(crate) struct ConnectedPeer {
     encoding: Option<Encoding>,
 }
 
+#[derive(Default)]
+pub(crate) struct ConnectedPeers(ArcSwap<im::HashMap<PeerId, ConnectedPeer>>);
```

Comment:
probably want to move ConnectedPeer+ConnectedPeers into a dedicated file in a follow-up -- feeels like a self-contained concept.

Reply:
done